com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/StackFrameManager.java

```diff
@@ -24,21 +24,47 @@ public class StackFrameManager implements IStackFrameManager {
     private Map<Long, StackFrame[]> threadStackFrameMap = Collections.synchronizedMap(new HashMap<>());
```

Comment:
If all the methods are synchronized, will we get benefit if we simply put a non-synced HashMap here?